src/library/dao/autodjcratesdao.cpp

```diff
@@ -10,6 +10,11 @@
 #include "library/trackcollection.h"
 #include "library/dao/autodjcratesdao.h"
 
+
+static const int kPreferredPercent = 25;
+// This requires that there are enough tracks in library such that
+// kPrefered_Percent of them is a sufficiently large no.
```

Comment:
Sorry , I didn't understand this . Are you saying I should rename the variable ?